### ISSUE TITLE
ssl

### ISSUE THREAD

**Issue by elaineo** [elaineo/etherwallet#1]
Add ssl cert for classicetherwallet.com

**Comment by RashawnSanchez** [comment]
You can use Let's Encrypt, it's free and easy to setup

**Comment by elaineo** [comment]
yeah, it's just that I'm using the same server to host multiple apps, so I have to rearrange the port fwding.

**Comment by matthewjamesr** [comment]
Let's encrypt is great, but does not verify identify technically. Thoughts?

**Comment by elaineo** [comment]
Now that it has a more permanent domain name, I'm happy to get a cert from a legitimate CA

**Comment by mikeyb** [comment]
Let's encrypt is just as legit as your standard $10 unverified cert. I wouldn't waste the money.  If you think users need an identity verified cert then go for it, but its a lot more cost than a standard unverified cert.   We are just looking for encrypted communication right? $0.02

**Comment by ericremoreynolds** [comment]
Thanks for your great work @elaineo!

Have you made any progress on the certificate? Right now chrome is giving me a net::ERR_INSECURE_RESPONSE error when checking the balance in the "Replay Protection" section.

**Comment by elaineo** [comment]
@ericremoreynolds this repo has moved to https://github.com/ethereumproject/etherwallet
Please post issues there
